src/hypothesis/internal/debug.py

```diff
@@ -17,14 +17,17 @@
 
 from __future__ import division, print_function, absolute_import
 
-from hypothesis import settings as Settings
-from hypothesis.core import find
+import os
+import sys
```

Comment:
Since you’ve deferred importing several other things until you use them, why not `sys`?